templates/base.html

```diff
@@ -149,7 +149,7 @@
                   {% trans "Your Cart" context "Main navigation item" %}
                 </span>
                 <div class="navbar__brand__cart__icon">
-                  <svg data-src="{% static "images/cart.svg" %}" width="35" height="30"/>
+                  <svg data-src="{% static "images/cart.svg" %}" width="35" height="30" alt="Items"/>
```

Comment:
I think that according to the good practices of writing alt texts we should be more descriptive here and say rather something like "An icon of a shopping cart." or even "An icon of a shopping cart possibly with the number of items added to the cart".

Reply:
I think in this particular case the icon is purely decorative as the textual label besides it does a good job describing what the button does. Decorative elements should have an empty alt.

Reply:
Plus, the label right next to the icon is saying "Your cart" then we have the cart's item count badge; it's already really descriptive and comprehensible without any alt text, even for blind users' accessibility.

<p align='center'><img src='https://user-images.githubusercontent.com/6186720/46897314-1f3d9500-ce82-11e8-9d15-6e2c51b1fd17.png' alt='screenshot' /></p>

Reply:
I'm not sure what's the question? You can edit the file and then commit. If you want to clean the commit history, you can rewrite the history using `git reset HEAD~1 --soft` (I would only recommend if you are used to `git` though), then commit over it and force push.